bindings/python/PySequence.cpp

```diff
@@ -230,7 +229,7 @@ Sequence_setAnimationData(Sequence* self, PyObject* args)
 			return nullptr;
 		}
 
-		data.push_back(value);
```

Comment:
Good catch! The fix however should just be to change `data.resize` to `data.reserve` which was the intention here. Same for `PySequenceEntry.cpp`.